scripts/launch_dev.sh

```diff
@@ -44,10 +48,10 @@ sudo docker run --rm -it --name eightknot_server \
 
 # cleanup
 printf "\nShutting down worker pool...\n"
-sudo docker stop worker_pool;
+docker stop worker_pool;
```

Comment:
why no sudo? (on all of these)

Reply:
I learned that it's considered bad practice to have docker containers elevated to super-user privilege if not necessary. It's also uncool to make the user use 'sudo' in a script that they didn't write.